core/variant_call.cpp

```diff
@@ -1654,6 +1655,7 @@ void register_variant_methods() {
 	ADDFUNC0R(DICTIONARY, ARRAY, Dictionary, keys, varray());
 	ADDFUNC0R(DICTIONARY, ARRAY, Dictionary, values, varray());
 	ADDFUNC1R(DICTIONARY, DICTIONARY, Dictionary, duplicate, BOOL, "deep", varray(false));
+	ADDFUNC2R(DICTIONARY, NIL, Dictionary, get, NIL, "key", NIL, "default", varray(Variant()));
```

Comment:
I'm fairly new to working with Variant, but isn't the `m_ret` argument set to `NIL` here when it shouldn't be? See [Line 1410](https://github.com/godotengine/godot/blob/c89bc047ef2f12e4acee4b025de74bb83a8fee62/core/variant_call.cpp#L1410).

Reply:
I'm also new to working with Variant, but in this case I think the `NIL` is acting as an "ANY Variant".

Example:
https://github.com/godotengine/godot/blob/c89bc047ef2f12e4acee4b025de74bb83a8fee62/core/variant_call.cpp#L1671-L1672